docs/griptape-framework/data/extraction-engines.md

```diff
@@ -0,0 +1,94 @@
+# Overview 
+Extraction Engines are used to extract data from text. 
+
+## CSV Extraction Engine
+
+Used to extract data from text using a CSV file. You can set a custom [prompt driver](../../griptape-framework/structures/prompt-drivers.md) or [chunker](../../griptape-framework/data/chunkers.md). 
```

Comment:
Can you please explain what you mean by "using a CSV file"?

Reply:
it should be "Used to extract data from a CSV file"